### DISCUSSION TITLE
After sucwessful deployment iam getting this sending error to any mail adress i try too send an email to

### DISCUSSION THREAD

**Discussion by Bugspiderlee** [postalserver/postal#2335] [Help with using Postal]
No SMTP servers were available for gmail.com. Tried gmail-smtp-in.l.google.com, alt1.gmail-smtp-in.l.google.com, alt2.gmail-smtp-in.l.google.com, alt3.gmail-smtp-in.l.google.com, and alt4.gmail-smtp-in.l.google.com

sending an emai to a gmail email or any other server

**Comment by willpower232** [comment]
This usually means that your server provider is blocking your outbound SMTP connections (or gmail is blocking your incoming connections). You should probably confirm with your server provider whether they are blocking SMTP.

**Comment by Bugspiderlee** [comment]
iam not so ure they are blocking iam using a digital ocean vps and this is happenning to all servers i try to send an emai to and the error is indicating SOftfail?

**Reply by willpower232** [reply]
Postal just sees a network issue which could be temporary but the chance of you not being able to reach gmail is almost nothing and this exact error message has cropped up a few times here when their server providers are blocking the ports.

I think digitalocean does block it on new accounts so you should get in touch with them to clarify the situation for your account in particular

**Reply by rafael-leo-almeida** [reply]
Is there any way to change outgoing SMTP port to 587 instead default 25?

**Reply by willpower232** [reply]
this is not currently a feature and I'm also not sure whether it is a good idea as you can't guarantee that port will be open on the receiving end

**Reply by Bugspiderlee** [reply]
@willpower232 Thanks for the feedback so what service providers can i use like public once which dont have smtp problems.
the once you will recommend.

**Reply by willpower232** [reply]
We're currently using AWS but we did have to complete some forms to allow us to send email so it is not totally straight forward. I'm not sure what other options there are as I haven't done any research into it.
